boa/src/object/internal_methods.rs

```diff
@@ -51,31 +73,67 @@ impl GcObject {
     ///
     /// [spec]: https://tc39.es/ecma262/#sec-ordinary-object-internal-methods-and-internal-slots-preventextensions
     #[inline]
-    pub fn prevent_extensions(&mut self) -> bool {
+    pub fn prevent_extensions(&mut self, context: &Context) -> Result<bool> {
+        self.ordinary_prevent_extensions(context)
+    }
+
+    /// Disable extensibility for ordinary object.
+    ///
+    /// More information:
+    ///  - [ECMAScript reference][spec]
+    ///
+    /// [spec]: https://tc39.es/ecma262/#sec-ordinarypreventextensions
+    #[inline]
+    pub fn ordinary_prevent_extensions(&mut self, _context: &Context) -> Result<bool> {
         self.borrow_mut().extensible = false;
-        true
+        Ok(true)
+    }
+    /// Delete a property.
```

Comment:
Leave an empty line for clarity
```suggestion
    }
    
    /// Delete a property.
```